tests/providers/amazon/aws/transfers/test_s3_to_redshift.py

```diff
@@ -420,16 +477,8 @@ def test_using_redshift_data_api(self, mock_rs, mock_run, mock_session, mock_con
             ),
         )
         op.execute(None)
-        copy_query = """
-                        COPY schema.table
-                        FROM 's3://bucket/key'
-                        credentials
-                        'aws_access_key_id=aws_access_key_id;aws_secret_access_key=aws_secret_access_key'
-                        ;
-                     """
+
         mock_run.assert_not_called()
-        assert access_key in copy_query
-        assert secret_key in copy_query
 
         mock_rs.execute_statement.assert_called_once()
         # test with all args besides sql
```

Comment:
Same about this comment as well. Only for non-obvious or cheeky things can we keep comments.